2-collectors/scala-stream-collector/examples/config.hocon.sample

```diff
@@ -34,100 +36,135 @@ collector {
     name = {{doNotTrackCookieName}}
+    name = ${?COLLECTOR_DO_NOT_TRACK_COOKIE_NAME}
     value = {{doNotTrackCookieValue}}
+    value = ${?COLLECTOR_DO_NOT_TRACK_COOKIE_VALUE}
   }
 
   # When enabled and the cookie specified above is missing, performs a redirect to itself to check
   # if third-party cookies are blocked using the specified name. If they are indeed blocked,
   # fallbackNetworkId is used instead of generating a new random one.
   cookieBounce {
     enabled = false
+    enabled = ${?COLLECTOR_COOKIE_BOUNCE_ENABLED}
     # The name of the request parameter which will be used on redirects checking that third-party
     # cookies work.
     name = "n3pc"
+    name = ${?COLLECTOR_COOKIE_BOUNCE_NAME}
     # Network user id to fallback to when third-party cookies are blocked.
     fallbackNetworkUserId = "00000000-0000-4000-A000-000000000000"
+    fallbackNetworkUserId = ${?COLLECTOR_COOKIE_BOUNCE_FALLBACK_NETWORK_USER_ID}
     # Optionally, specify the name of the header containing the originating protocol for use in the
     # bounce redirect location. Use this if behind a load balancer that performs SSL termination.
     # The value of this header must be http or https. Example, if behind an AWS Classic ELB.
     forwardedProtocolHeader = "X-Forwarded-Proto"
+    forwardedProtocolHeader = ${?COLLECTOR_COOKIE_BOUNCE_FORWARDED_PROTOCOL_HEADER}
   }
 
   # When enabled, the redirect url passed via the `u` query parameter is scanned for a placeholder
   # token. All instances of that token are replaced withe the network ID. If the placeholder isn't
   # specified, the default value is `${SP_NUID}`.
   redirectMacro {
     enabled = false
+    enabled = ${?COLLECTOR_REDIRECT_MACRO_ENABLED}
     # Optional custom placeholder token (defaults to the literal `${SP_NUID}`)
     placeholder = "[TOKEN]"
+    placeholder = ${?COLLECTOR_REDIRECT_REDIRECT_MACRO_PLACEHOLDER}
   }
 
   # Customize response handling for requests for the root path ("/").
   # Useful if you need to redirect to web content or privacy policies regarding the use of this collector.
   rootResponse {
     enabled = false
+    enabled = ${?COLLECTOR_ROOT_RESPONSE_ENABLED}
     statusCode = 302
+    statusCode = ${?COLLECTOR_ROOT_RESPONSE_STATUS_CODE}
     # Optional, defaults to empty map
     headers = {
-        Location = "https://127.0.0.1/",
-        X-Custom = "something"
+      Location = "https://127.0.0.1/",
+      Location = ${?COLLECTOR_ROOT_RESPONSE_HEADERS_LOCATION},
+      X-Custom = "something"
     }
     # Optional, defaults to empty string
     body = "302, redirecting"
+    body = ${?COLLECTOR_ROOT_RESPONSE_BODY}
+  }
+
+  # Configuration related to CORS preflight requests
+  cors {
+    # The Access-Control-Max-Age response header indicates how long the results of a preflight
+    # request can be cached. -1 seconds disables the cache. Chromium max is 10m, Firefox is 24h.
+    accessControlMaxAge = 5 seconds
```

Comment:
Ideally each newly introduced variable should have the ${?ENV_VARIABLE} counterpart coming with it, do you agree?

Reply:
done, thanks :+1: